frontend/src/metabase/palette/components/PaletteResults.tsx

```diff
@@ -28,6 +29,12 @@ export const PaletteResults = () => {
     [results],
   );
 
+  useEffect(() => {
+    if (processedResults[0] === t`Search results`) {
+      query.setActiveIndex(2);
+    }
+  }, [processedResults, query]);
```

Comment:
it's not obvious to me what or why this is being done? can you name add some naming / comments that help provide some context?

Reply:
Super fair. I'll stick this in a function with a clear name and a comment.